deployments/fargate/README.md

```diff
@@ -0,0 +1,95 @@
+# AWS Fargate Deployment
+Familiarity with AWS Fargate (Fargate) is assumed. Consult the 
+[User Guide for AWS Fargate](https://docs.aws.amazon.com/AmazonECS/latest/userguide/what-is-fargate.html) for further reading.
+
+The
+[Splunk OpenTelemetry Collector](https://github.com/signalfx/splunk-otel-collector)
+(splunk-collector or collector) can be deployed to an ECS Task as a standalone container 
+or as sidecar container alongside the application containers that you want to monitor.
+
+Find the Docker image repository for the collector
+[here](https://quay.io/repository/signalfx/splunk-otel-collector?tab=tags).
+The latest tag is `quay.io/signalfx/splunk-otel-collector:latest`.
+
+## Standalone Container
+As a standalone container, you add the container definition for the collector to a
+task that is separate from the task containing the application containers that you 
+want to monitor. You configure the collector to use extension
+[Amazon Elastic Container Service Observer (ecsobserver)](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/extension/observer/ecsobserver#amazon-elastic-container-service-observer).
+to discover targets in running tasks, filtered by service names, task definitions
+and container labels. The ecsobserver is currently limited to discovering
+prometheus targets. Also, the task role must include the read-only permissions below. 
+The permissions can be added to a customer-managed policy that is attached to the
+task role.
+
+```text
+ecs:List*
+ecs:Describe*
+```
+
+Below is an example of a collector configuration that configures extension
```

Comment:
Suggested change: "The following is an example collector configuration showing the ecs_observer (ecsobserver) extension configured to to find Prometheus targets in the splunk-fargate-cluster cluster in the us-west-2 region with the NGINX_PROMETHEUS_EXPORTER_PORT container label. ecs_observer then writes the results to file /etc/ecs_sd_targets.yaml, which the Prometheus receiver is configured to read."

Reply:
I edited the readme entirely. Please see later commit.